src/elements/content-preview/ContentPreview.js

```diff
@@ -682,12 +711,14 @@ class ContentPreview extends PureComponent<Props, State> {
             showDownload: this.canDownload(),
             skipServerUpdate: true,
             useHotkeys: false,
+            enableThumbnailsSidebar,
         };
         const { Preview } = global.Box;
         this.preview = new Preview();
         this.preview.addListener('load', this.onPreviewLoad);
         this.preview.addListener('preview_error', this.onPreviewError);
         this.preview.addListener('preview_metric', this.onPreviewMetric);
+        this.preview.addListener('viewerevent', this.onPreviewViewerEvent);
```

Comment:
You should be able to add these at the top level if you want.
this.preview.addListener('thumbnailsOpen', this.onThumbnailsOpen); etc...

Reply:
I'll do that, seems better than adding the `onPreviewerEvent` for just these two